src/utils.js

```diff
@@ -58,6 +58,57 @@ export function getRootExpression(node) {
 }
 
 /*
+ * Returns the name of the reference node passed.
+ *
+ * example: var foo = document;
+ *  The node for foo will return 'document'
```

Comment:
We can get this via the `Identifier` event using ESLint, so why all the context lookup again? https://github.com/mozilla/addons-validator/blob/master/src/rules/javascript/mozindexeddb.js#L5